components/support/ktx/src/main/java/mozilla/components/support/ktx/kotlin/String.kt

```diff
@@ -190,6 +190,26 @@ fun String.getDataUrlImageExtension(defaultExtension: String = "jpg"): String {
         .find(this)?.groups?.get(1)?.value ?: defaultExtension
 }
 
+/**
+ * Returns the input string preceded by 4 ellipses, locked LTR, to be used for
+ * obfuscating credit card numbers.
+ */
+fun String.addEllipsesToCreditCardNumber(): String {
```

Comment:
I thought our intention was to add a getter in the `CreditCard` data class to return the obfuscated card number? This seems ok, but maybe too specific for CreditCards. I will check in with some AC folks to see if there are any strong opinions.

Reply:
I agree with @gabrielluong,  this function is too spesific to `CreditCard`  to be as `String` extension function. If we don't want to add the function directly to `CreditCard` we would make this to be an extension function of `CreditCard`.

Reply:
Will request changes then.

Reply:
@gabrielluong @Amejia481 I agree. I do want to add a getter, but there is more than one `crediCard` type object, as you mentioned, and I did not want to have the same code in two places. Should I just add the custom getter to both?

Reply:
I think either option is fine. The `CreditCard` in concept-engine is more specific to the GV Autocomplete `CreditCard`, which will have the credit card decrypted at that point. You will also need to get the last 4 digits for that one. So, it seems easier for me to imagine 2 different getters due to the additional need to get the last 4 digits for the concept-engine `CreditCard`. I don't really know what a custom getter for both will look like, so feel free to post both version and we can work through it together.